### PR TITLE
Add label so type includes `repeated`

### PR DESCRIPTION
**Pull request by Breefield**
See: https://github.com/protobuffet/docusaurus-protobuffet/issues/6

### CODE REVIEW

File: packages/docusaurus-protobuffet-plugin/src/theme/ProtoFile.tsx [hunk 1]
```diff
@@ -144,6 +144,9 @@ const ProtoMessageFields = (props: MessageFieldsProps) => {
     );
 
     return (
+      { if(field.label != "") {
```
Comment (AnthonyBobsin) [inline packages/docusaurus-protobuffet-plugin/src/theme/ProtoFile.tsx:147]: there are some code formatting issues here that will break things. can we conditionally inject this in a different way?

**Review by AnthonyBobsin — CHANGES_REQUESTED** [review]
I don't think this will render based on code formatting issues. Please test, let me know if you need assistance.

**Comment by Breefield** [comment]
Hey @AnthonyBobsin I think I've got this written a little more cleanly/without syntax errors (I hope, I don't write TSX).

**Comment by Breefield** [comment]
Hey @AnthonyBobsin, would you like any additional changes?

**Comment by AnthonyBobsin** [comment]
@Breefield thanks for addressing the changes! I tested it out and found a minor issue with the rendering.

image

For types with links, the label value is included in the link. I think we should change this behaviour to exclude the label value from the link.

Here is a code snippet applying the suggested behaviour. Please apply these changes and I'll merge after that fix.


```tsx
const FieldTypeCell = ({ field }: { field: MessageField}) => {
  const typeCell = field.typeLink === undefined
    ? <code>{field.longType}</code>
    : <Link to={field.typeLink}><code>{field.longType}</code></Link>;

  return (
    <span>
      {field.label !== "" && <i>{field.label}&nbsp;</i>}
      {typeCell}
    </span>
  )
}
```

image

**Comment by Breefield** [comment]
@AnthonyBobsin modified!

**Comment by AnthonyBobsin** [comment]
@Breefield Thanks for adding this feature! I've released it under version 0.3.3.

**Pull request merged**